AdminApi.Tests.Integration/StepDefinitions/IdentitiesApiStepDefinitions.cs

```diff
@@ -5,16 +5,17 @@ namespace AdminApi.Tests.Integration.StepDefinitions;
 
 [Binding]
 [Scope(Feature = "GET Identities")]
-public class IdentitiesApiStepDefinitions : BaseStepDefinitions<List<IdentitySummaryDTO>>
+public class IdentitiesApiStepDefinitions : BaseStepDefinitions
 {
     private readonly IdentitiesApi _identitiesApi;
+    private HttpResponse<List<IdentitySummaryDTO>> _response;
 
-    public IdentitiesApiStepDefinitions(IdentitiesApi identitiesApi) : base(new HttpResponse<List<IdentitySummaryDTO>>())
+    public IdentitiesApiStepDefinitions(IdentitiesApi identitiesApi) : base()
     {
         _identitiesApi = identitiesApi;
+        _response = new HttpResponse<List<IdentitySummaryDTO>>();
```

Comment:
We shouldn't initialize the `_response` from the constructor. Doesn't make sense.
Diogo probably did it to avoid the warning the compiler gives about an uninitialized field. But just setting it to a plain object is a lie. 
You should rather make `_response` optional and use `?`/`!` wherever it is used.